packages/application-components/src/components/public-page-layout/public-page-layout.tsx

```diff
@@ -56,12 +719,29 @@ const PublicPageLayoutContent: FC<TProps> = (props) => {
 };
 
 const PublicPageLayout: FC<TProps> = (props) => {
+  // JK: Do this in development
+  //const enableWorkspacesUi = { value: true };
+  // JK: Do this for the real thing
+  const enableWorkspacesUi = useFlagVariation(
+    featureFlags.ENABLE_WORKSPACES_UI
+  );
+  const isWorkspacesUiEnabled =
+    // @ts-ignore In case it's coming from the MC API, it's an object { value: boolean }.
+    enableWorkspacesUi?.value ?? enableWorkspacesUi;
```

Comment:
I believe we can omit the fallback value since we've defined this only as as an API flag, right?

Reply:
yes good point